.ci/Jenkinsfile

```
@@ -237,14 +237,18 @@ pipeline {
         }
       }
     }
+    // This meta-stage happens in the internal-ci instance to be able to sign the artifacts correctly.
     stage('Release') {
       options {
         skipDefaultCheckout()
         timeout(time: 12, unit: 'HOURS')
       }
       when {
         beforeAgent true
-        tag pattern: 'v\\d+.*', comparator: 'REGEXP'
+        allOf {
+          tag pattern: 'v\\d+.*', comparator: 'REGEXP'
+          expression { env.JENKINS_URL.contains('internal-ci') }
```

https://github.com/elastic/apm-pipeline-library/pull/727

```suggestion
          expression { isInternalCI() }
```